### DISCUSSION TITLE
Rich Text Editor 🧾 💡

### DISCUSSION THREAD

**Discussion by codenashwan** [wireui/wireui#567] [Ideas]
👋 I'm looking for a good rich text editor for Laravel, especially for the Livewire framework. Please try to add this unique feature because people are really tired of using similar rich text editors like Tinymce, CKEditor, FCKeditor, etc. for config it to livewire

<img width="637" alt="Screenshot 2022-12-11 at 2 31 32 AM" src="https://user-images.githubusercontent.com/35005761/206879413-27e4ba22-baf5-47dc-a01a-a0f89afa998f.png">

**Comment by tinusg** [comment]
CKEditor and its alternatives are the result of tens of thousands of man hours. This is not a feature that can simply be added to a random project.

**Comment by ph7jack** [comment]
It would be nice, and I have plans to create it.
However, it's not a high priority. 
You can use the [QuillJS](https://quilljs.com/), it's easy and awesome.
